src/strategies/roundrobin.js

```diff
@@ -0,0 +1,16 @@
+
+module.exports = class RoundRobinStrategy {
+
+	constructor() {
+		this.counter = 0;
+	}
+
```

Comment:
Would be good an init base method (like in [other components](https://github.com/WoLfulus/moleculer/blob/a0d6f24e9dd3fe475f56564ba46421b5238b29fd/src/transporters/base.js#L37)), which will be called by ServiceRegistry. It will pass the `broker` to this function so you will have a pointer to `broker` or `nodes` list...etc